elasticdl/pkg/ps/checkpoint.go

```diff
@@ -112,10 +118,10 @@ func LoadModelFromCheckPoint(checkPointDir string, shardID int, shardNum int) (*
 	return model, nil
 }
 
-// SaveModelToCheckPoint saves in-memory model to checkpoint
-func SaveModelToCheckPoint(checkPointDir string, model *Model, shardID int, shardNum int) {
-	os.MkdirAll(checkPointDir, os.ModePerm)
+// SaveModelToCheckpoint saves in-memory model to checkpoint
+func SaveModelToCheckpoint(checkpointDir string, model *Model, shardID int, shardNum int) {
+	os.MkdirAll(checkpointDir, os.ModePerm)
```

Comment:
We may need to check whether the `checkpointDir` exists.

Reply:
From the document, https://golang.org/pkg/os/#MkdirAll, "If path is already a directory, MkdirAll does nothing and returns nil."